src/h5web/visualizations/line/LineVis.tsx

```diff
@@ -17,14 +18,12 @@ interface Props {
 function LineVis(props: Props): ReactElement {
   const { dataArray } = props;
 
-  const [showGrid, hasYLogScale] = useLineConfig(
-    (state) => [state.showGrid, state.hasYLogScale],
+  const [showGrid, scaleType] = useLineConfig(
+    (state) => [state.showGrid, state.scaleType],
     shallow
   );
 
-  const dataDomain = useMemo(() => {
-    return findDomain(dataArray.data as number[]);
-  }, [dataArray]);
+  const dataDomain = useAxisDomain(dataArray, scaleType);
```

Comment:
This was made a single hook rather than put in the `LineConfig` as I had an issue where the domain of the `LineConfig` was out-of-phase with the data displayed. This led to at best, an additional render and at worst, a WebGL crash...